content/blog/pycon-india-2016-workshop-introduction.md

```diff
@@ -0,0 +1,33 @@
+Title: PyCon India 2016 - Workshop Introduction
+Date: 2016-08-17 15:00
+Category: 2016
+Slug: workshop-introduction-2016
+Summary: PyCon India 2016 - Workshop Introduction
+
+
+The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016 and are two and half hour long of hands-on sessions, so be sure not to miss it.
+
```

Comment:
The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016. They are hands-on sessions of 2.30hr duration each and are taught by experts in the field. You don't want to miss them.